app/imports/api/sprints/schema.js

```diff
@@ -7,6 +7,29 @@ const SprintSchema = new SimpleSchema({
     projectId: {
         type: String,
     },
+    closed: {
+        type: Boolean,
+        defaultValue: false,
+    },
+});
+
+// Methods schema
+
+
+export const SprintsIdentitySchema = new SimpleSchema({
+    sprintId: {
+        type: String,
+    },
+});
+
+export const AddSprintSchema = new SimpleSchema({});
```

Comment:
Here you have schemas for methods.

`addSprint` methods have two argument - `name`, `projectId`. In this schema by extending it you have 4 required arguments.

This schema should take care only about `name` and `projectId`.